src/contracts.ts

```diff
@@ -1,11 +1,16 @@
 import { PartialReadonlyContractAbi } from './types'
+import { AbiType, MethodAbi, EventAbi } from 'web3'
 
-export const getMethod = (abi: PartialReadonlyContractAbi, name: string) => {
-  return abi.filter(x => x.type == 'function' && x.name == name)[0]
+export const getMethod = (abi: PartialReadonlyContractAbi, name: string): MethodAbi => {
+  // Have to cast since there's a bug in
+  // web3 types on the 'type' field
+  return abi.filter(x => x.type == AbiType.Function && x.name == name)[0] as MethodAbi
```

Comment:
use `find(...)`, not `filter(...)[0}`

Reply:
changing to use `find` but there's still this annoying thing due to a bad internal lib i think:
<img width="556" alt="Screen Shot 2019-08-15 at 1 50 26 PM" src="https://user-images.githubusercontent.com/1011391/63114809-c0deb880-bf63-11e9-9fb4-670c6410a28e.png">